octobot/strategy_optimizer/strategy_optimizer.py

```diff
@@ -239,12 +245,15 @@ def print_report(self):
         self.logger.info(f"{best_result[CONFIG].get_result_string()} "
                          f"average trades count: {best_result[TRADES_IN_RESULT]:f}")
 
-    def get_overall_progress(self):
-        return int((self.run_id - 1) / self.total_nb_runs * 100) if self.total_nb_runs else 0
+    async def get_overall_progress(self):
+        return int((self.run_id - 1) / self.total_nb_runs * 100) if self.total_nb_runs else 0, 0
 
-    def is_in_progress(self):
+    async def is_in_progress(self):
```

Comment:
async ?

Reply:
to keep the same interface as StrategyDesignOptimizer